src/include/cc.h

```diff
@@ -83,12 +85,38 @@ struct Pcb {
   }
   void rto_advance() { rto_timer++; }
 
+  void shift_right_sack_bitmap() {
+    // Since we have 4 elements of 64 bits in the SACK bitmap we start from 3
+    // The value 3 will have to change if we go beyond 256 bits.
+    // For instance if we have 512 bits we will have 8 elements of 64 bits,
+    // In that case we will have to start from 7.
+    for (int i = 3; i > 0; --i) {
```

Comment:
Replace the magic constant `3` with something clearer, like `sizeof(sack_bitmap)/8`.